garnbarn_api/tests/test_user_model.py

```diff
@@ -0,0 +1,24 @@
+from django.test import TestCase, testcases
+from garnbarn_api.models import CustomUser
+
+
+class TestCustomUserModel(TestCase):
+    def setUp(self):
+        """Create setup for user model."""
+        self.user1 = CustomUser(uid='test1', line="line_id1")
+        self.user2 = CustomUser(uid='test2', line="line_id2")
+        self.user3 = CustomUser(line="line_id3")
```

Comment:
Can the model require the field `uid`? Since we don't want anyone to create the record without the uiod

Reply:
The field is required. But its default value is "Unknown" as we discussed last time.

Reply:
Maybe changing it to be a bare string is better?